src/Routing/Filter/AssetFilter.php

```diff
@@ -118,8 +117,7 @@ protected function _deliverAsset(Request $request, Response $response, $assetFil
 			$response->header('Content-Length', filesize($assetFile));
 		}
 		$response->cache(filemtime($assetFile));
-		$response->send();
-		ob_clean();
+		$response->sendHeaders();
 		readfile($assetFile);
 		if ($compressionEnabled) {
 			ob_end_flush();
```

Comment:
@ADmad when zlib.output_compression in php.ini is off ,response is failed [screenshot](https://www.dropbox.com/s/zkoz5ykdp388p3t/Screenshot%202015-01-07%2022.50.00.png?dl=0)
but is on it is ok [screenshot](https://www.dropbox.com/s/qmq4ajdsdl6ddg7/Screenshot%202015-01-07%2022.56.29.png?dl=0)
Is it possible to check this problem?